scripts/vote_2023_10_31.py

```diff
@@ -0,0 +1,114 @@
+"""
+todo: change title
+Voting 31/10/2023.
+
+"""
+
+import time
+
+from typing import Dict
+from brownie.network.transaction import TransactionReceipt
+from brownie import interface
+from utils.voting import bake_vote_items, confirm_vote_script, create_vote
+from utils.agent import agent_forward
+from utils.ipfs import upload_vote_ipfs_description, calculate_vote_ipfs_description
+from utils.easy_track import add_evmscript_factory, create_permissions, remove_evmscript_factory
+
+from utils.config import (
+    get_deployer_account,
+    contracts,
+    get_is_live,
+    get_priority_fee
+)
+
+# todo: change description
+description = """
+### Omnibus on-chain vote contains 3 motions:
+
+1. Switch off TopUp ETs for RCC PML ACT
+2. Switch on TopUpStable ETs for RCC PML ACT
+
+"""
+
+def start_vote(tx_params: Dict[str, str], silent: bool) -> bool | list[int | TransactionReceipt | None]:
+    """Prepare and run voting."""
+
+    rcc_dai_topup_factory_old = interface.IEVMScriptFactory("0x84f74733ede9bFD53c1B3Ea96338867C94EC313e")
+    pml_dai_topup_factory_old = interface.IEVMScriptFactory("0x4E6D3A5023A38cE2C4c5456d3760357fD93A22cD")
+    atc_dai_topup_factory_old = interface.IEVMScriptFactory("0x67Fb97ABB9035E2e93A7e3761a0d0571c5d7CD07")
+
+    # todo: change addresses
+    rcc_stable_topup_factory = interface.TopUpAllowedRecipients("0x84f74733ede9bFD53c1B3Ea96338867C94EC313e")
+    pml_stable_topup_factory = interface.TopUpAllowedRecipients("0x4E6D3A5023A38cE2C4c5456d3760357fD93A22cD")
+    atc_stable_topup_factory = interface.TopUpAllowedRecipients("0x67Fb97ABB9035E2e93A7e3761a0d0571c5d7CD07")
+
+    # todo: change addresses
+    rcc_stable_registry = interface.AllowedRecipientRegistry("0xDc1A0C7849150f466F07d48b38eAA6cE99079f80")
+    pml_stable_registry = interface.AllowedRecipientRegistry("0xDFfCD3BF14796a62a804c1B16F877Cf7120379dB")
+    atc_stable_registry = interface.AllowedRecipientRegistry("0xe07305F43B11F230EaA951002F6a55a16419B707")
+
+    call_script_items = [
+        # 1. Switch off TopUp ETs for RCC PML ACT
```

Comment:
There should be finance limits update:
https://github.com/lidofinance/scripts/blob/33034fe893a190991ee26e30c3118789feb5dedd/archive/scripts/vote_2022_12_06_1.py#L61-L102

see: https://discord.com/channels/761182643269795850/1163831729635733574/1166262759630061588
and https://github.com/lidofinance/lido-improvement-proposals/blob/develop/LIPS/lip-13.md